dnfdaemon-client/commands/install/install.hpp

```diff
@@ -33,7 +33,7 @@ class CmdInstall : public TransactionCommand {
     void run(Context & ctx) override;
 
 private:
-    libdnf::OptionBool * strict_option{nullptr};
+    libdnf::OptionBool strict_option {false};
```

Comment:
Are you sure that about formatting? I suggest to use it without the space `libdnf::OptionBool strict_option{false};`